### DISCUSSION TITLE
How to synchronize atoms?

### DISCUSSION THREAD

**Discussion by AjaxSolutions** [pmndrs/jotai#1508] [Ideas] [answered]
I wish Jotai provided a build-in utility to synchronize atoms something similar to Recoil Atom Effects.

I'd like to observe changes on an atom and then update other atoms.

**Comment by dai-shi** [comment, marked as answer]
> I'd like to observe changes on an atom and then update other atoms.

It's really bad practice. You should derive atoms or update two atoms at the same time.

**Comment by dmaskasky** [comment]
https://jotai.org/docs/extensions/effect
